src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -341,10 +325,6 @@ def query_top_tags(
             query=filter_query,
             params=params,
             orderby=orderby,
-            conditions=[
-                [translated_aggregate_column, "IS NOT NULL", None],
-                ["tags_key", "IN", [tag_key]],
-            ],
```

Comment:
oh this confused me for a second. I see we no longer pass `conditions` since they will always be ignored because we're only be using `extra_snql_condition`

Reply:
Oh good point I should rename the variables :+1: